tests/integration/test_plugins_usgs.py

```diff
@@ -129,6 +168,49 @@ def test_usgs_monitoring_features(query, expected_count):
     assert count == expected_count
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("query, expected_count", [
+    ({"parent_features": ['USGS-02020004'], "feature_type": "point"}, 52)],
+                         ids=[ "points_by_subbasin"])
+def test_usgs_monitoring_features2(query, expected_count):
+    """Test USGS search by region  """
+
+    synthesizer = register(['basin3d.plugins.usgs.USGSDataSourcePlugin'])
+    monitoring_features = synthesizer.monitoring_features(**query)
+
+    # TODO should there be some kind of exeption handling for invalid queries that don't return anything?
+    count = 0
+    for mf in monitoring_features:
+        count += 1
+        print(
+            f"{mf.id} ({mf.feature_type}) {mf.description} {mf.coordinates and [(p.x, p.y) for p in mf.coordinates.absolute.horizontal_position]}")
+        if 'feature_type' in query:
+            assert mf.feature_type == query['feature_type'].upper()
+
+    assert count == expected_count
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("query, expected_count", [({"monitoring_features": ["USGS-09129600"], "feature_type": "point"}, 1)],
+                         ids=["point_by_id"])
+def test_usgs_monitoring_features3(query, expected_count):
+    """Test USGS search by region  """
+
+    synthesizer = register(['basin3d.plugins.usgs.USGSDataSourcePlugin'])
+    monitoring_features = synthesizer.monitoring_features(**query)
+
+    # TODO should there be some kind of exeption handling for invalid queries that don't return anything?
+    count = 0
+    for mf in monitoring_features:
+        count += 1
+        print(
+            f"{mf.id} ({mf.feature_type}) {mf.description} {mf.coordinates and [(p.x, p.y) for p in mf.coordinates.absolute.horizontal_position]}")
+        if 'feature_type' in query:
+            assert mf.feature_type == query['feature_type'].upper()
+
+    assert count == expected_count
+
+
```

Comment:
Why are these new tests included? It looks like they are already included in the parameterization for `test_usgs_monitoring_features` (starting in new line 153).

Reply:
@dschristianson they are there because it's easier to test the two outliers (point by id and point by subbasin) individually. It's not really needed, I can remove it.

Reply:
@dschristianson removed the 2 individual tests from the "main" `test_usgs_monitoring_features` to facilitate easier testing

Reply:
@heycatwonton: you can isolate the parameterized test for testing -- we can show you. I prefer to keep them all in the main `test_usgs_monitoring_features` unless there is something fundamentally different in the test.